packages/calling-stateful-client/src/CallSubscriber.ts

```diff
@@ -160,10 +160,18 @@ export class CallSubscriber {
 
   private remoteParticipantsUpdated = (event: { added: RemoteParticipant[]; removed: RemoteParticipant[] }): void => {
     event.added.forEach((participant: RemoteParticipant) => {
-      this.addParticipantListener(participant);
+      try {
+        this.addParticipantListener(participant);
+      } catch (e) {
+        console.error(e);
```

Comment:
We shouldn't have to `console` log the error here.

The real mystery is why the exception that escaped here wasn't bubbling all the way up and getting reported on console already.